llvm/unittests/TargetParser/Host.cpp

```diff
@@ -85,6 +85,9 @@ TEST(getLinuxHostCPUName, AArch64) {
   EXPECT_EQ(sys::detail::getHostCPUNameForARM("CPU implementer : 0x41\n"
                                               "CPU part        : 0xd0c"),
             "neoverse-n1");
+  EXPECT_EQ(sys::detail::getHostCPUNameForARM("CPU implementer : 0x41\n"
```

Comment:
You can commit this missing test separately.

Reply:
done. I've created a separate PR https://github.com/llvm/llvm-project/pull/106749